tests/test_game.py

```diff
@@ -78,3 +78,17 @@ def test_displays_the_board_and_a_message_on_game_over(players):
     game.game_over(console)
 
     assert console_io.print_output_call_count == 2
+
+
+def test_a_game_which_is_not_completed_remains_in_progress(game):
+    assert game.in_progress() is True
```

Comment:
Since `game.in_progress()` is boolean, could you drop the `is True`?

Reply:
👍